charts/longhorn/100.1.0+up1.2.2/Chart.yaml

```diff
@@ -5,7 +5,8 @@ annotations:
   catalog.cattle.io/namespace: longhorn-system
   catalog.cattle.io/os: linux
   catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
-  catalog.cattle.io/rancher-version: '>= 2.6.0-0'
+  catalog.cattle.io/rancher-version: '>= 2.6.0-0 <= 2.6.100-0'
+  catalog.cattle.io/kube-version: '>= 1.18.0-0'
```

Comment:
@shuo-wu left this one with an open range because I don't know if this chart supports 1.22. If it doesn't please let me know and I will add `< 1.22.0-0` to it.

Reply:
yes I believe this supports 1.22

Reply:
Longhorn v1.1.x does not support Kubernetes v1.22 while Longhorn v1.2 would support it.